build/vendor.ts

```diff
@@ -21,12 +23,14 @@ async function vendorDependencies() {
 
 	const indexHtmlPath = path.join(directoryOfThisFile, '..', 'app', 'index.html')
 	const oldIndexHtml = await fs.readFile(indexHtmlPath, 'utf8')
-	const importmap = dependencyPaths.reduce((importmap, { packageName, entrypointFile }) => {
-		importmap.imports[packageName] = `./${path.join('.', 'vendor', packageName, entrypointFile).replace(/\\/g, '/')}`
-		return importmap
-	}, { imports: {} as Record<string, string> })
-	const importmapJson = JSON.stringify(importmap, undefined, '\t')
-		.replace(/^/mg, '\t\t')
+	const importmap = dependencyPaths.reduce(
+		(importmap, { packageName, entrypointFile }) => {
+			importmap.imports[packageName] = `./${path.join('.', 'vendor', packageName, entrypointFile).replace(/\\/g, '/')}`
+			return importmap
+		},
+		{ imports: {} as Record<string, string> }
+	)
+	const importmapJson = JSON.stringify(importmap, undefined, '\t').replace(/^/gm, '\t\t')
```

Comment:
Why was this changed?  Removal of `es-module-shims` shouldn't have any impact on this file other than removing `es-module-shims` as a dependency.

Reply:
the editor formats them on save. reverting these lines to avoid confusion